api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/NewClaimServiceTest.java

```diff
@@ -72,8 +74,10 @@ void shouldSaveNonExistingEligibleClaimantAndSendNewCardMessage() {
         EligibilityResponse eligibilityResponse = anEligibilityResponse();
         given(client.checkEligibility(any())).willReturn(eligibilityResponse);
         given(eligibilityStatusCalculator.determineEligibilityStatus(any())).willReturn(EligibilityStatus.ELIGIBLE);
-        VoucherEntitlement voucherEntitlement = aValidVoucherEntitlement();
-        given(entitlementCalculator.calculateVoucherEntitlement(any(), any(), any())).willReturn(voucherEntitlement);
+        VoucherEntitlement firstVoucherEntitlement = aVoucherEntitlementWithEntitlementDate(LocalDate.now());
+        VoucherEntitlement secondVoucherEntitlement = aVoucherEntitlementWithEntitlementDate(LocalDate.now().plusWeeks(1));
+        var paymentCycleVoucherEntitlement = new PaymentCycleVoucherEntitlement(asList(firstVoucherEntitlement, secondVoucherEntitlement));
```

Comment:
Maybe not for this PR, but I still do think that the voucher entitlements should be stored within the `PaymentCycleVoucherEntitlement` in some kind of Map so that the `VoucherEntitlement`s are stored by date and more understandable.

Reply:
As discussed, we'll do this in the next PR.